### ISSUE TITLE
Lock Bios

### ISSUE THREAD

**Issue by jjodah** [MrChromebox/firmware#439]
HI, is there anyway to lock the bios after its installed?

**Comment by MrChromebox** [comment]
yes, but it's complicated and you'd need to build a custom firmware image that aligned properly to the regions on the flash chip which can be locked
